### ISSUE TITLE
what is the [random letters and numbers] mean?

### ISSUE THREAD

**Issue by unixc3t** [spymastermatt/thunderbird-monterail#84]
what is the [random letters and numbers].default mean?

**Comment by conema** [comment]
It literally means random letters and numbers, something like 12345xyz.default. Do you need help with the installation of the theme?

**Comment by lindhe** [comment]
I agree this should be clarified somehow. I understood it, but was also caught on it for a second.
